src/main/java/com/textadventure/Lexer.java

```diff
@@ -26,8 +25,11 @@ private TokenType(String pattern) {
 			tokenPatternsBuffer.append(String.format("|(?<%s>%s)", tokenType.name(), tokenType.pattern));
```

Comment:
We can rename the function to `parse(String input)` since that makes more sense than `lex` which means lexical.